src/dbnode/storage/stats/query_stats_default_tracker.go

```diff
@@ -53,27 +60,53 @@ func DefaultQueryStatsTracker(
 		MetricsScope().
 		SubScope("query-stats")
 	return &queryStatsTracker{
-		options:    queryStatsOpts,
-		recentDocs: scope.Gauge("recent-docs-per-block"),
-		totalDocs:  scope.Counter("total-docs-per-block"),
+		options:                   queryStatsOpts,
+		recentDocs:                scope.Gauge("recent-docs-matched"),
+		totalDocs:                 scope.Counter("total-docs-matched"),
+		recentBytesRead:           scope.Gauge("recent-bytes-read"),
+		totalBytesRead:            scope.Counter("total-bytes-read"),
+		recentDocsLimitError:      scope.Tagged(map[string]string{"limit": "docs-matched"}).Counter("error"),
+		recentBytesReadLimitError: scope.Tagged(map[string]string{"limit": "bytes-read"}).Counter("error"),
```

Comment:
nit: these aren't really errors, rather limits being exceeded

Reply:
Will change to `limit-exceeded`